### PR TITLE
Uniformizing indentation all over RISC-V zkVM .asm file.

### PR DESCRIPTION
**Pull request by lvella**
@georgwiese complained about indentation in the output .asm file, so this PR is his fault.

### CODE REVIEW

File: riscv/src/compiler.rs [hunk 1]
```diff
@@ -200,44 +200,50 @@ pub fn compile(
...
+        program.push((2, "x1 <== jump(__data_init);".to_string()));
     }
     program.extend([
-        format!("// Set stack pointer\nx2 <=X= {stack_start};"),
-        "x1 <== jump(__runtime_start);".to_string(),
-        "return;".to_string(), // This is not "riscv ret", but "return from powdr asm function".
+        (2, "// Set stack pointer".to_string()),
+        (2, format!("x2 <=X= {stack_start};")),
+        (2, "x1 <== jump(__runtime_start);".to_string()),
+        (2, "return;".to_string()), // This is not "riscv ret", but "return from powdr asm function".
```
Comment (leonardoalt) [inline riscv/src/compiler.rs:231]: I'm not sure all this weirdness (2, ...) everywhere is worth it

**Comment by leonardoalt** [comment]
Instead of outputting formatted code, wouldn't it make more sense to output whatever and have a separate formatter? Don't we already have one for PIL?

File: riscv/src/compiler.rs (hunk 1 above)
Comment (lvella) [reply riscv/src/compiler.rs:231]: I thought about it halfway through. Anyway, this 2 is the indentation level.

Comment (leonardoalt) [reply riscv/src/compiler.rs:231]: I got that, but I don't think it's a good solution.

**Comment by lvella** [comment]
We close it.

**Pull request closed without merging**